vagrant/Makefile

```diff
@@ -101,3 +101,38 @@ create-secrets:
 # patch context to use sumologic namespace as default
 patch-context:
 	kubectl config set-context --current --namespace "${namespace}"
+
```

Comment:
Is there no need of `nginx-ingress-k8s` now as there were commands for this as well in the old Makefile

Reply:
Good point. We are decided to support official nginx for now, so there is no need to install the kubernetes' one. Anyway, I will prepare separate PR for that